fw-child/resources/js/child-functions.js

```diff
@@ -191,10 +191,10 @@ const $ = jQuery;
 
     // SMOOTH SCROLL
 
-    let scrollOffset = 0;
+    let glossaryScrollOffset = 0;
 
     if ($('#glossary-list-nav').length) {
-      scrollOffset = $('#glossary-list-nav').outerHeight();
+      glossaryScrollOffset = $('#glossary-list-nav').outerHeight();
```

Comment:
No need to change because it's really not a big issue, but just note that the height of the nav is determined only once. If we resize the window and the nav becomes on two or three lines, it becomes taller, but the scroll offset won't be updated.

Reply:
You're right! We had no 2 or 3 lines nav at first when I coded this. I'll change it. Thanks :)